From 8d162bf6312a8f0b277131d1bf5b4292f3ddd50b Mon Sep 17 00:00:00 2001
From: Apekshit <apeksharma@gmail.com>
Date: Mon, 11 Jan 2016 12:04:53 -0800
Subject: [PATCH] HBASE-14159 Add dependency on
 hadoop-mapreduce-client-jobclient from hbase-spark to get rid of following
 error: [WARNING] warning: Class org.apache.hadoop.mapred.MiniMRCluster not
 found - continuing with a stub. [WARNING] one warning found (Apekshit)

---
 hbase-spark/pom.xml | 12 +++++++++---
 1 file changed, 9 insertions(+), 3 deletions(-)

diff --git a/hbase-spark/pom.xml b/hbase-spark/pom.xml
index 2efbca0..8f71a89 100644
--- a/hbase-spark/pom.xml
+++ b/hbase-spark/pom.xml
@@ -520,6 +520,12 @@
             <groupId>commons-logging</groupId>
             <artifactId>commons-logging</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.apache.hadoop</groupId>
+            <artifactId>hadoop-mapreduce-client-jobclient</artifactId>
+            <type>test-jar</type>
+            <scope>test</scope>
+        </dependency>
     </dependencies>
 
 
@@ -600,10 +606,10 @@
         </plugins>
     </build>
 
-  <profiles>
+    <profiles>
     <!-- Skip the tests in this module -->
-      <profile>
-        <id>skipSparkTests</id>
+        <profile>
+            <id>skipSparkTests</id>
             <activation>
                 <property>
                     <name>skipSparkTests</name>
-- 
2.3.2 (Apple Git-55)

